submissions/__init__.py

```diff
@@ -1,2 +1,2 @@
 """ API for creating submissions and scores. """
-__version__ = '3.1.12'
+__version__ = '3.2.1'
```

Comment:
Confirm numbering before merging (i.e. if this merges before the bump to `3.2.1`, make this the `3.2.0` version.

Reply:
Yep. I made this version in anticipation of Jansen's pr being merged before mine.